tests/test_text.py

```diff
@@ -0,0 +1,223 @@
+# -*- coding: utf-8 -*-
+# Copyright FMR LLC <opensource@fidelity.com>
+# SPDX-License-Identifier: GNU GPLv3
+import pandas as pd
+
+from feature.selector import Selective, SelectionMethod
+from test_base import BaseTest
+from textwiser import TextWiser, Embedding, PoolOptions, Transformation, WordOptions
+
+
+
+class TestText(BaseTest):
+
+    """
+        optimization_method : random
+        num_features:
+            - t = maximum number of features defines by user. The cost_metric input argument should ignore
+            - t = None: the number of feature computed by solving a set cover problem with cost metrics
+                (unicost or diverse)
+    """
+    def test_text_based_random_max_num_feature(self):
+        data = pd.DataFrame(
+            {"item1": ["this is a sentences with more common words and more words to increase frequency"],
+             "item2": ["second one in list of more frequent sentences with some repeated words"],
+             "item3": ["a word for more complexity and longer length"],
+             "item4": ["sentence with a lot of repeated common words and more words to increase frequency"],
+             "item5": ["more frequent words with a valid vector and more words to increase frequency"]})
+        labels = pd.DataFrame({"item1": [0, 1, 0, 0, 0], "item2": [1, 0, 0, 1, 0], "item3": [0, 0, 1, 0, 1],
+                               "item4": [0, 1, 1, 0, 1], "item5": [0, 1, 0, 0, 1]})
+
+        print(data)
+        print(labels)
+
+        method = SelectionMethod.TextBased(num_features=3,
+                                           optimization_method="random")
+
+        selector = Selective(method)
+        selector.fit(data, labels)
+        selector.transform(data)
```

Comment:
can we add some self.asserTrue() to check these outputs. Currerntly it just runs but does not verfiy anything in the output. See other tests in other classes and things they verify as the output

Reply:
same comment for other tests

Reply:
Sure, I added some assertations to verify that the output of the tests are correct and consistent.